src/index.ts

```diff
@@ -29,4 +43,44 @@ const run = async () => {
 	}
 };
 
+/* eslint-disable @typescript-eslint/naming-convention */
+const installCLI = async (): Promise<void> => {
+	const currentFile = url.fileURLToPath(import.meta.url);
+	const currentDir = path.dirname(currentFile);
+	const parentDir = path.resolve(currentDir, "..");
+
+	// Execute bash script
+	const cmdOut = await exec.getExecOutput(
+		`sh -c "` + parentDir + `/entrypoint.sh"`,
+	);
+
+	// Add path to 1Password CLI to $PATH
+	const outArr = cmdOut.stdout.split("\n");
+	if (outArr[0] && process.env.PATH) {
+		const cliPath = outArr[0]?.replace(/^(::debug::OP_INSTALL_DIR: )/, "");
+		core.addPath(cliPath);
+	}
+};
+
+const loadSecrets = async (shouldExportEnv: boolean) => {
+	// Pass User-Agent Inforomation to the 1Password CLI
```

Comment:
Typo: change from `Inforomation` to `Information`.

Reply:
Addressed in [`6ecd5ce`](https://github.com/1Password/load-secrets-action/pull/36/commits/6ecd5ceaf74eb507deb9fb2d36c18b03422431bc).